### PR TITLE
Update e2e testing scripts

### PR DESCRIPTION
**Pull request by jleaniz**
### Description of the change

Updates to e2e test scripts.

### Applicable issues

- fixes #

### Checklist

- [ ] All tests were successful.
- [ ] Unit tests added.
- [ ] Documentation updated.

### CODE REVIEW

File: turbinia/e2e/run-e2e-gke.sh [hunk 1]
```diff
@@ -79,23 +80,36 @@ turbinia-client config list
...
-# Wait until request is received
-req=$(turbinia-client status request $REQUEST_ID -j)
-while [[ -z "$req" ]]
+# Wait until request is running
+req_status=$(turbinia-client status request $REQUEST_ID -j | jq -r '.status')
+RETRIES=0
+while [[ $req_status != "running" ]]
 do
-  echo "Request $REQUEST_ID is still populating. Sleeping for 5 seconds..."
+  RETRIES-=1
```
Comment (wajihyassine) [inline turbinia/e2e/run-e2e-gke.sh:88]: Should this be `RETRIES+=1` seems like this would go negative?

File: turbinia/e2e/run-e2e-gke.sh [hunk 2]
```diff
@@ -79,23 +80,36 @@ turbinia-client config list
...
 # Wait until request is complete 
-req_status=$(turbinia-client status request $REQUEST_ID -j | jq -r '.status')
+RETRIES=0
 while [[ $req_status = "running" ]]
 do
   req_status=$(turbinia-client status request $REQUEST_ID -j | jq -r '.status')
   if [[ $req_status = "running" ]]
   then
-    echo "Turbinia request $REQUEST_ID is still running. Sleeping for 180 seconds..."
+    RETRIES-=1
```
Comment (wajihyassine) [inline turbinia/e2e/run-e2e-gke.sh:106]: Same as above, should this be `RETRIES+=1`

**Review by wajihyassine — COMMENTED** [review]
Thanks for fixing, overall LGTM minus one nit about the incrementing

File: turbinia/e2e/run-e2e-gke.sh (hunk 1 above)
Comment (jleaniz) [reply turbinia/e2e/run-e2e-gke.sh:88]: Doh. Yes, thanks Wajih :)

**Review by wajihyassine — APPROVED** [review]
LGTM!

**Pull request merged**